conjure-java-core/src/main/java/com/palantir/conjure/java/services/JerseyServiceGenerator.java

```diff
@@ -187,10 +187,15 @@ private MethodSpec generateServiceMethod(
                     .build());
         }
 
-        endpointDef
-                .getDeprecated()
-                .ifPresent(
-                        deprecatedDocsValue -> methodBuilder.addAnnotation(ClassName.get("java.lang", "Deprecated")));
+        endpointDef.getDeprecated().ifPresent(_deprecatedValue -> {
```

Comment:
this code is now duplicated in 3 places - undertow, jersey, and dialogue.  Happy to add it to a util class if we want.